Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm

```diff
@@ -831,8 +833,15 @@ static void validateClass(WKRemoteObjectDecoder *decoder, Class objectClass)
     if (objectClass == [NSInvocation class] || objectClass == [NSBlockInvocation class])
         return;
 
-    if (![decoder validateClassSupportsSecureCoding:objectClass])
-        [NSException raise:NSInvalidUnarchiveOperationException format:@"Object of class \"%@\" does not support NSSecureCoding.", objectClass];
+    @try {
+        if (![decoder validateClassSupportsSecureCoding:objectClass])
+            [NSException raise:NSInvalidUnarchiveOperationException format:@"Object of class \"%@\" does not support NSSecureCoding.", objectClass];
+    } @catch(NSException *exception) {
+        NSString *message = [NSString stringWithFormat:@"%@,%@", NSStringFromClass(objectClass), exception];
+        std::array<uint64_t, 6> values { 0, 0, 0, 0, 0, 0 };
+        strncpy(reinterpret_cast<char*>(values.data()), message.UTF8String, sizeof(values));
+        CRASH_WITH_INFO(values[0], values[1], values[2], values[3], values[4], values[5]);
```

Comment:
For the future, it might be nice to extract this into an inline method or extract it into its own `CRASH()` macro that takes a C-string (since this is the second and third copy of this code, IIRC).  Not required to fix for this PR.